.github/workflows/ci-cd.yml

```
@@ -0,0 +1,102 @@
+        
+    - name: Prepare server for deployment
+      uses: appleboy/ssh-action@v1.0.3
+      with:
+        host: ${{ secrets.SERVER_HOST }}
+        username: ${{ secrets.SERVER_USER }}
+        password: ${{ secrets.SERVER_SSH_KEY }}
+        script: |
+          mkdir -p /opt/bilbomusicbot
+          cd /opt/bilbomusicbot
+          docker pull ${{ secrets.DOCKER_USERNAME }}/bilbomusicbot:latest
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)